.spelling

```diff
@@ -53,7 +53,6 @@ admissionregistration.k8s.io
 AES-NI
 AKS
 Alibaba
-ALLOW_ANY
```

Comment:
for my own education, we don't have to put technical words here if we enclose them in backticks?

Reply:
Correct. Backticks render as `code blocks`. Symbols are treated as a code blocks through the docs.